x-pack/plugins/infra/public/components/asset_details/content/content.tsx

```diff
@@ -35,6 +35,16 @@ export const Content = ({
       <TabPanel activeWhen={FlyoutTabIds.ANOMALIES}>
         <Anomalies nodeName={node.name} onClose={overrides?.anomalies?.onClose} />
       </TabPanel>
+      <TabPanel activeWhen={FlyoutTabIds.OVERVIEW}>
+        <Overview
+          currentTimeRange={currentTimeRange}
+          nodeName={node.name}
+          nodeType={nodeType}
+          onTabsStateChange={(tabId: TabIds) => onChange({ activeTabId: tabId })}
+          dataView={overrides?.overview?.dataView}
+          dateRange={overrides?.overview?.dateRange!}
```

Comment:
Let's pass the `dateRange`  as is and define a default date range in the `Overview` component
```suggestion
          dateRange={overrides?.overview?.dateRange}
```

Perhaps 15m range as default would work. wdyt?

Reply:
Sounds good, this is temporary as I already mentioned -  until we have the date picker in the tab so for the time until that this is fine (then we will remove that from the tab and prefill the date picker and rely on it - also we can have the same concept with the default there as well)